doc/compliancetest.rst

```diff
@@ -203,6 +203,31 @@ left of the "PowerOutlet" text in the tool bar. The symbol to the right of the
 "PowerOutlet" text shows a green check mark when the outputs are on, and a
 black cross when the outputs are off (or when the power outlet not is connected).
 
+Hardware naming
+^^^^^^^^^^^^^^^
+For the conformance test is different type of Siemens hardware used.
```

Comment:
This sentence structure is weird. "Different types of Siemens hardware are used for the conformance test."

Reply:
Done